perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ServicesManagerMethod.java

```diff
@@ -399,7 +399,7 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 
 		@Override
 		public Service call(ApiCaller ac, Deserializer parms) throws PerunException {
-			return ac.getServiceById(parms.readInt("id"));
+			return ac.getServicesManager().getServiceById(ac.getSession(), parms.readInt("id"));
```

Comment:
Its strange, that this method is not allowed for facility admins (same for ByName version), but facility admins can retrieve all services together. We should allow it for them in both methods.